### PR TITLE
docs: update shared cloud docs

### PR DESCRIPTION
**Pull request by bmorelli25**
@EamonnTP pointed out in https://github.com/elastic/observability-docs/issues/144 that the ESS shared docs content is outdated. This PR updates it.

In addition, this PR adds a new getting started snippet for Observability. I'm not sure if the best approach here is to have two different files, or to pepper a bunch of `ifdef::[]` and `ifndef::[]` statements into one cloud getting started snippet. Thoughts?

Closes https://github.com/elastic/observability-docs/issues/144.

### CODE REVIEW

File: shared/cloud/ess-getting-started-obs.asciidoc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Include this file in your docs:
+// include::{docs-root}/shared/cloud/ess-getting-started-obs.asciidoc[]
+
+// To include APM Server instructions, add this attribute:
+// :include-apm-instructions:
+
+. {ess-trial}[Get a free trial].
+
+. Log into {ess-console}[Elastic Cloud].
```
Comment (dedemorton) [inline shared/cloud/ess-getting-started-obs.asciidoc:9]:
```suggestion
. Log into {ess-console}[{ecloud}].
```

File: shared/cloud/ess-getting-started-obs.asciidoc [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+. Click *Create deployment*.
+
+. Select *Elastic Observability* and give your deployment a name.
+
+ifdef::include-apm-instructions[]
+. Click *Create deployment* and copy the password for the `elastic` user.
+
+. Select *APM* from the menu on the left and make note of the APM endpoint and APM Server secret token.
+You'll need these in step two.
```
Comment (dedemorton) [inline shared/cloud/ess-getting-started-obs.asciidoc:19]: I'm just doing a quick drive-by review, but I wonder if it's a good idea to give a specific step number here (steps can move around or be removed over time, and people forget to update references like this one.

**Review by dedemorton — COMMENTED** [review]
Couple of minor comments. Note that I didn't verify the steps, but assume you did. :-)

File: shared/cloud/ess-getting-started-obs.asciidoc (hunk 2 above)
Comment (bmorelli25) [reply shared/cloud/ess-getting-started-obs.asciidoc:19]: Ahh! Good catch. That's a copy/pasta from my other PR

Comment (bmorelli25) [reply shared/cloud/ess-getting-started-obs.asciidoc]: Fixed. Thanks!

File: shared/cloud/ess-getting-started.asciidoc [hunk 3]
```diff
@@ -6,12 +6,14 @@
...
+. {ess-trial}[Get a free trial].
 
 . Log into {ess-console}[Elastic Cloud].
 
-. Click *Create deployment* and give your deployment a name.
-. Optional: The default options are great for getting started, but you can add more features by clicking *Customize deployment*.
-. Click *Create deployment* and copy down the password for the `elastic` user and Cloud ID information.
+. Click *Create deployment*.
+
+. Select a deployment type and give your deployment a name.
```
Comment (nrichers) [inline shared/cloud/ess-getting-started.asciidoc:15]:
```suggestion
. Select a solution and give your deployment a name.
```

Milestone 45 brought with it support for our 3+1 solutions. Technically, it's not wrong to say "deployment type" but the UI does refer to solutions. It might be helpful to stick with consistent terminology. 

image

**Review by nrichers — APPROVED** [review]
LGTM! One minor wording tweak you can commit if you agree with the change. Thank you so much for updating this shared content! 🚀

File: shared/cloud/ess-getting-started.asciidoc (hunk 3 above)
Comment (bmorelli25) [reply shared/cloud/ess-getting-started.asciidoc:15]: Great point!
